addons/website_slides/models/slide_channel.py

```diff
@@ -501,12 +508,14 @@ def _add_groups_members(self):
         for channel in self:
             channel._action_add_members(channel.mapped('enroll_group_ids.users.partner_id'))
 
-    def _remove_membership(self, partner_ids):
+    def _remove_membership(self, partner_ids, unlink_slide_partners=False):
```

Comment:
add explanation in docstring about the new param (for which kind of cases it's used)